src/ElasticSearchIndexer.php

```diff
@@ -200,7 +200,11 @@ private function extendBatch(string $action, string $indexName, IndexDocument $d
             return $params;
         }
 
-        $body = array_merge($document->getBody(), (array)$document->getDynamicProperties());
+        $body = array_merge(
```

Comment:
Could you please add tests?

Reply:
Added test for the ElasticSearchIndexer class.